polygeist/lib/Dialect/Polygeist/Transforms/LICM.cpp

```diff
@@ -703,16 +703,44 @@ void AffineParallelGuardBuilder::getConstraints(
     }
 }
 
+//===----------------------------------------------------------------------===//
+// LICMCandidate
+//===----------------------------------------------------------------------===//
+
+class LICMCandidate {
+public:
+  LICMCandidate(Operation &op) : op(op) {}
+  Operation &getOp() const { return op; }
```

Comment:
Change name `getOp` to `getOperation` (I see that `OperationSideEffects` in the same file uses that term` already).